### DISCUSSION TITLE
CONPOT MOD BUS PORT 502 not Work

### DISCUSSION THREAD

**Discussion by juancagomez** [telekom-security/tpotce#1708] [Q&A]
I have version 24.04.1 of TPOT installed and in the CONPOT docker I do not see the MODBUS port 502 up and operational, how can I make it operational?

Host is up (0.093s latency).

PORT    STATE  SERVICE
502/udp closed mbap

Nmap done: 1 IP address (1 host up) scanned in 0.96 seconds

**Comment by t3chn0m4g3** [comment]
Besides 24.04.1 is not even released yet, we do not have the conpot_default profile enabled which supports modbus (since most of the ports are occupied otherwise).
You can however follow the `docker-compose.yml` for [conpot](https://github.com/telekom-security/tpotce/blob/master/docker/conpot/docker-compose.yml) guiding you how the profile works and adjust `$HOME/tpotce/docker-compose.yml` accordingly.

**Comment by ncristiz** [comment]
Lo resolviste ?. Did you solve it ?.
